include/envoy/server/admin.h

```diff
@@ -115,17 +115,15 @@ class Admin {
    * Executes an admin request with the specified query params. Note: this must
    * be called from Envoy's main thread.
    *
-   * @param path the path of the admin URL.
-   * @param param the query-params passed to the admin request handler.
+   * @param path_and_query the path and query of the admin URL.
```

Comment:
I'm not sure how this snuck in here - looks like there's some files from this change and an rbac change?